scripts/config.js

```diff
@@ -1,2 +1,94 @@
+  },
+  {
+    name: 'Demis Hassabis',
+    img: 'img/demis-hassabis.jpg',
+    highlight: 'Artificial Intelligence researcher, entrepreneur, and UK Government AI Advisor',
+    description: 'Chief executive and co-founder of DeepMind and Insomorphic Labs',
+  },
+  {
+    name: 'Fei-Fei Li',
+    img: 'img/fei-fei-li.jpg',
+    highlight: 'Sequoia Capital Professor of Computer Science at Standford University',
+    description: 'Established ImageNet, the dataset that enabled rapid advances in computer vision in the 2010s',
+  },
+  {
+    name: 'Cynthia Breazeal',
+    img: 'img/cynthia-breazeal.jpg',
+    highlight: 'Professor of media arts and science at MIT',
+    description: 'Founded and directs the Personal Robots group at MIT\'s Media Lab',
+  },
+  {
+    name: 'Raymond Kurzweil',
+    img: 'img/raymond-kurzweil.jpg',
+    highlight: 'Computer scientist, author, investor, and futurist',
+    description: 'Involved in fields such as optical character recognition, text-to-speech synthesis, and speech recognition technology',
+  },
+  {
+    name: 'Jurgen Schmindhuber',
+    img: 'img/jurgen-schmidhuber.jpg',
+    highlight: 'Renown in the fields of artificial intelligence, deep learning and artificial neural networks',
+    description: 'Co-director of the Dalle Molle Institute for Artificial Intelligence Research in Lugano, Switzerland',
+  },
+];
+
+// --------- Header Variables ---------------------------
+const mobileMenu = document.querySelector('.mobile-menu');
+const openMobileBtn = document.querySelector('.open-mobile');
+const closeMobileBtn = document.querySelectorAll('.close-mobile');
+
+// --------- Speakers Section Variables -----------------
+const speakersSection = document.getElementById('featured-speakers');
+
+// --------- Functions ----------------------------------
+
+const openMobileMenu = () => {
+  mobileMenu.classList.remove('hidden');
+  openMobileBtn.classList.add('hidden');
+};
+
+const closeMobileMenu = () => {
+  mobileMenu.classList.add('hidden');
+  openMobileBtn.classList.remove('hidden');
+};
+
+const renderSpeakersSection = () => {
+  speakersSection.classList.add('py-8', 'text-center', 'sm:pb-16', 'xl:pt-12', 'xl:pb-24', 'xl:min-h-[100vh]');
+
+  const speakersContent = `<h2 class="text-xl font-semibold xl:text-3xl">Featured Speakers</h2>
+    <div class="flex justify-center">
+        <i class="las la-minus text-2xl text-[#ec5242]"></i>
+        <i class="las la-minus text-2xl text-[#ec5242] ml-[-0.5rem]"></i>
+    </div>
+
+    <div class="speakers-grid pl-2 pr-3 grid grid-cols-1 grid-rows-6 gap-8 md:grid-cols-2 md:grid-rows-3 md:gap-x-2 md:px-12 lg:px-36 xl:mt-8 xl:px-60 xl:gap-y-24">
+    </div>`;
+
+  speakersSection.insertAdjacentHTML('beforeend', speakersContent);
+
+  const speakersGrid = document.querySelector('.speakers-grid');
+
+  speakersData.forEach((speaker) => speakersGrid.insertAdjacentHTML('beforeend',
+    `<div class="speaker-card relative flex gap-16 md:gap-20 lg:gap-24 xl:gap-20">
```

Comment:
Kindly use semantic article tag instead of div for each speaker card to make your web page more semantic and SEO friendly.